manifests/charts/base/templates/crds.yaml

```diff
@@ -1,7 +1,4 @@
 {{- if .Values.base.enableCRDTemplates }}
 {{ .Files.Get "crds/crd-all.gen.yaml" }}
 {{ .Files.Get "crds/crd-operator.yaml" }}
-{{ .Files.Get "crds/maistra.io_meshfederations.yaml" }}
-{{ .Files.Get "crds/maistra.io_serviceexports.yaml" }}
-{{ .Files.Get "crds/maistra.io_servicemeshextensions.yaml" }}
 {{- end }}
```

Comment:
@rcernich Should we keep https://github.com/jwendell/istio-maistra/blob/MAISTRA-2330/manifests/charts/base/templates/role.yaml#L22 in this repo? Or delete it from here as well and move it to operator?